gensim/models/wrappers/ldamallet.py

```diff
@@ -36,6 +36,8 @@
 
 import numpy
 
+from bs4 import BeautifulSoup
```

Comment:
Using Beautiful soup for parsing correct XML is an overkill. Please use `from xml.etree import ElementTree`